shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/mysql/server-5.yaml

```diff
@@ -31,7 +31,7 @@ mode:
 rules:
   - !AUTHORITY
     users:
-      - root@:Root@123
+      - proxy@:Proxy@123
```

Comment:
Are these files under shardingsphere-integration-test-scaling still used? You have copied them to test-env.

Reply:
there are some other case still need these config files. I don't want to make this PR too huge .